src/module/fluid.js

```diff
@@ -38,9 +38,11 @@ var upPath = path.resolve(__dirname, "../../../../..");
 var upInfusionPath = resolveModuleSync("infusion", upPath);
 if (upInfusionPath) {
     upInfusion = require(upInfusionPath);
-    upInfusion.log("Resolved infusion from path " + __dirname + " to " + upInfusion.module.modules.infusion.baseDir);
-    module.exports = upInfusion;
-    return;
+    if (upInfusion && Object.keys(upInfusion).length != 0) {
+        upInfusion.log("Resolved infusion from path " + __dirname + " to " + upInfusion.module.modules.infusion.baseDir);
+        module.exports = upInfusion;
+        return;
+    } else { console.log(upInfusionPath + " resolved to current Infusion"); }
```

Comment:
I'm guessing this might also fail ESLinting because of a lack of line breaks around the block?